sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -415,39 +415,72 @@ opentitan_functest(
     test_harness = "//sw/host/tests/rom/e2e_bootstrap_disabled",
 )
 
-opentitan_functest(
-    name = "e2e_chip_specific_startup",
-    srcs = ["chip_specific_startup.c"],
-    args = [],
-    cw310 = cw310_params(
-        test_cmds = [
-            "--rom-kind=rom",
-            "--bitstream=\"$(location //hw/bitstream:rom)\"",
-            "--bootstrap=\"$(location {flash})\"",
-            "--otp-unprogrammed",
+[otp_image(
+    name = "otp_img_chip_specific_startup_{}".format(lc_state.lower()),
+    src = "//hw/ip/otp_ctrl/data:otp_json_{}".format(lc_state.lower()),
+    overlays = STD_OTP_OVERLAYS,
+) for lc_state in structs.to_dict(CONST.LCV)]
```

Comment:
Aren't these the same OTP images as `otp_img_e2e_bootstrap_entry_*` above, and thus create the same set of bitstream splices?

Should we be using a single set of common OTPs and splices for varying the lifecycle states?

Reply:
You are correct. Originally, I created a separate image set because I thought there was going to have to be an additional overlay (created in a follow up PR) because of [this](https://github.com/lowRISC/opentitan/issues/14493#issuecomment-1316106323). But after looking into it further, these values are already set in the STD overlays so we should be all set. Good catch! Only the test itself needs to be fixed up.

Fixed.